daffodil-lib/src/main/scala/edu/illinois/ncsa/daffodil/util/DecimalUtils.scala

```diff
@@ -0,0 +1,328 @@
+/*
```

Comment:
Would changing this file and object PackedUtils or PackedDecimalUtils or something else be more appropriate and help clarify that it's only intended for these packed formats?

Reply:
I was planning on also adding conversion functions for Zoned Decimal in here too, so I just went with DecimalUtils.